packages/desktop-client/src/components/modals/OutOfSyncMigrationsModal.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { Trans } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+
+import { closeBudget } from 'loot-core/client/actions';
+
+import { Button } from '../common/Button2';
+import { Link } from '../common/Link';
+import { Modal, ModalHeader, ModalTitle } from '../common/Modal';
+import { Paragraph } from '../common/Paragraph';
+import { Text } from '../common/Text';
+import { View } from '../common/View';
+
+export function OutOfSyncMigrationsModal() {
+  const dispatch = useDispatch();
+
+  const closeBudgetAndModal = (close: () => void) => {
+    dispatch(closeBudget());
+    close();
+  };
+
+  return (
+    <Modal name="out-of-sync-migrations">
+      {({ state: { close } }) => (
+        <>
+          <ModalHeader title={<ModalTitle title="Please update Actual!" />} />
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
          <ModalHeader title={<ModalTitle title={t('Please update Actual!')} />} />
```